skill-tree /src/main/resources/logback-test.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
Should we not have a file appender as well so that we generate log files?
Do we generate log files for other services?

Reply:
Currently we don't have log files for the other services, the main backend is hosted on https://railway.app/